From 6b635f399d4af2370ec44440f3237ab4242d032d Mon Sep 17 00:00:00 2001
From: Lionel Xu <Lionel.Xu@freescale.com>
Date: Fri, 2 Mar 2012 13:07:57 +0800
Subject: ENGR00170526-2 AUDIO: Remove struct field rst_gpio to audio platform
 data

Remove rst_gpio field to mxc_audio_platform_data.

Signed-off-by: Lionel Xu <Lionel.Xu@freescale.com>
---
 include/linux/fsl_devices.h |    1 -
 1 file changed, 1 deletion(-)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index 849d4e4..72544b3 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -353,7 +353,6 @@ struct mxc_audio_platform_data {
 	int hp_active_low;	/* headphone irq is active loaw */
 
 	int sysclk;
-	int rst_gpio;
 	const char *codec_name;
 
 	int (*init) (void);	/* board specific init */
-- 
1.7.9.5

